src/Components/SmartComponents/Reports/Common/tablePage.js

```diff
@@ -3,13 +3,14 @@ import PropTypes from 'prop-types';
 import { View, Text, Link } from '@react-pdf/renderer';
 import { Table } from '@redhat-cloud-services/frontend-components-pdf-generator';
 import { processDate } from '@redhat-cloud-services/frontend-components-utilities/helpers';
+import { parseRhelVersions } from '../../../../Helpers/VulnerabilityHelper';
 import { CVES_PATH } from '../../../../Helpers/constants';
 import messages from '../../../../Messages';
 import styles from './styles';
 import { formatDate } from '../../../../Helpers/MiscHelper';
 import CVElabels from '../Common/CVElabels';
 
-const tablePage = ({ data, page, intl, header, type, isReportDynamic = false }) => {
+const tablePage = ({ data, page, intl, header, type, isReportDynamic = false, isAppliesToOSEnabled }) => {
     const POSSIBLE_HEADERS = [
```

Comment:
When toggling the (preview) feature flag it looks like the column is still shown, but is gone when refreshing. I think we will need to wrap this in a `useMemo` and make it dependent on the feature flag.

Reply:
Great suggestion! Applied and works perfectly now ❤️